app/(main)/league/[leagueId]/entry/[entryId]/week/WeekTeams.tsx

```diff
@@ -67,6 +68,13 @@ const WeekTeams = ({
               <WeeklyPickButton
                 team={competition.team.name}
                 src={competition.team.logo}
+                isDisabled={
+                  selectedTeams.find(
```

Comment:
Create a helper function for this. Keeping our UI components presentational, will allow use to test functionality like this outside of the component.